docs/source/markdown/podman-rm.1.md

```diff
@@ -42,7 +42,8 @@ The storage option conflicts with the **--all**, **--latest**, and **--volumes**
 
 **--volumes**, **-v**
 
-Remove the volumes associated with the container.
+Clean-up volumes created specifically for the container. This does not include volumes created
+with `podman volume create`, or the `--volume` option of `podman run` and `podman create`.
```

Comment:
Not entirely accurate - it includes anonymous volumes created with `--volume`.

Reply:
@mheon thank you for pointing it out